pkg/broker/ingress/handler.go

```diff
@@ -47,6 +47,9 @@ const (
 	// TODO(liu-cong) configurable timeout
 	decoupleSinkTimeout = 30 * time.Second
 
+	// Limit for request payload in bytes (100Mb)
+	maxRequestBodyBytes = 100000000
```

Comment:
Let me know if we'd rather have this as an env variable.

Reply:
Can you document how this number is picked? Is this the same as the pubsub message size limit?

Reply:
Sure, I have updated the description.